settings.json

```diff
@@ -0,0 +1,600 @@
+                            "default": 300
+                        },
+                        "OPTIONS": {
+                            "type": "object",
+                            "properties": {
+                                "CLIENT_CLASS": {
+                                    "type": "string",
+                                    "default": "django_redis.client.DefaultClient"
+                                },
+                                "PASSWORD": {
+                                    "type": "string",
+                                    "default": ""
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "SECRET_KEY": {
+            "type": "string",
+            "default": ""
+        },
+        "BANNER_TOP": {
+            "type": "string",
+            "default": ""
+        },
+        "BANNER_BOTTOM": {
+            "type": "string",
+            "default": ""
+        },
+        "BANNER_LOGIN": {
+            "type": "string",
+            "default": ""
+        },
+        "BRANDING_FILE_PATH": {
+            "type": "object",
+            "properties": {
+                "logo": {
+                    "type": "string",
+                    "default": ""
+                },
+                "favicon": {
+                    "type": "string",
+                    "default": ""
+                },
+                "icon_16": {
+                    "type": "string",
+                    "default": ""
+                },
+                "icon_32": {
+                    "type": "string",
+                    "default": ""
+                },
+                "icon_180": {
+                    "type": "string",
+                    "default": ""
+                },
+                "icon_192": {
+                    "type": "string",
+                    "default": ""
+                },
+                "icon_mask": {
+                    "type": "string",
+                    "default": ""
+                },
+                "header_bullet": {
+                    "type": "string",
+                    "default": ""
+                },
+                "nav_bullet": {
+                    "type": "string",
+                    "default": ""
+                }
+            }
+        },
+        "BRANDING_PREPENDED_FILENAME": {
+            "type": "string",
+            "default": "nautobot_"
+        },
+        "BRANDING_TITLE": {
+            "type": "string",
+            "default": "Nautobot"
+        },
+        "BRANDING_URLS": {
+            "type": "object",
+            "properties": {
+                "code": {
+                    "type": "string",
+                    "default": "https://github.com/nautobot/nautobot"
+                },
+                "docs": {
+                    "type": "string",
+                    "default": "<STATIC_URL>docs/index.html"
```

Comment:
Wonder if there's a way for us to make this dynamically correct. Maybe we could treat `settings.json` as a Jinja2 template or something and render it with the appropriate `static_url` as part of the REST endpoint?

Reply:
I suppose similarly we could dynamically sub in in the environment variables as defaults for all of the settings that support environment variable configuration, etc.? Not sure if that's needed or not.